src/scraping/collector.py

```
@@ -0,0 +1,123 @@
+                    "location": listing.get("location", {}).get("address"),
+                    "price": listing.get("price", 0),
+                    "type": listing.get("propertyType"),
+                    "features": listing.get("features", []),
+                    "listing_age": listing.get("daysOnMarket", 0),
+                    "property_value": listing.get("estimatedValue", 0),
+                    "location_score": self._calculate_location_score(listing),
+                    "market_trend": self._get_market_trend(listing)
+                }
+                leads.append(lead)
+            except Exception as e:
+                print(f"Error processing listing: {str(e)}")
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)